syntaxes/weidu-tp2.tmLanguage.yml

```diff
@@ -1235,6 +1235,7 @@ repository:
       - match: (%SCROLL_WIZARD_TIME_STOP%)
       - match: (%SCROLL_WIZARD_TROLLISH_FORTITUDE%)
       - match: (%SCROLL_WIZARD_TRUE_DISPEL_MAGIC%)
+      - match: (%SCROLL_WIZARD_DISPEL_MAGIC%)
```

Comment:
This list is imported from IElib, too.
The change is not harmful, but it will not persist. Will get overwritten on next import if it's not in IElib.

Reply:
> This list is imported from IElib, too. The change is not harmful, but it will not persist. Will get overwritten on next import if it's not in IElib.

Removed.